src/app/properties/property.service.ts

```diff
@@ -44,11 +44,15 @@ export class PropertyService {
 
   getUnapprovedProperties(searchDto?: PropertiesSearchDto): Observable<Property[]> {
     let url = `${environment.apiHost}/${ApiPaths.UnapprovedProperties}`;
-
-
     return this.http.get<Property[]>(url);
   }
 
+  getMyProperties(hostId: any): Observable<Property[]> {
+    let url = `${environment.apiHost}/${ApiPaths.Properties}/${ApiPaths.MyProperties}`;
```

Comment:
This endpoint does not seem to exist on the backend.

Reply:
> It needs some updates on back-end to fully work.

This explains it.

Reply:
Ok, sorry about that.